mysql-test/mroonga/storage/column/timestamp/fractional_seconds/t/null.test

```diff
@@ -0,0 +1,43 @@
+# -*- mode: sql; sql-product: mysql -*-
+#
+# Copyright (C) 2024  Kodama Takuya <otegami@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+--source ../../../../../include/mroonga/have_mroonga.inc
+--source ../../../../../include/mroonga/load_mroonga_functions.inc
+
+--disable_warnings
+DROP TABLE IF EXISTS diaries;
+--enable_warnings
+
+CREATE TABLE diaries (
+  name VARCHAR(255),
+  created_at TIMESTAMP(1) NULL,
+  KEY created_at_index(created_at)
+) DEFAULT CHARSET=utf8mb4;
+
+INSERT INTO diaries VALUES ("zero day", "0000-00-00 00:00:00.0");
+INSERT INTO diaries VALUES ("null day", NULL);
+INSERT INTO diaries VALUES ("Mroonga release day", "2010-8-19 00:00:00.0");
```

Comment:
"time" is better than "day" because type is `TIMESTAMP`.

Reply:
fix: [62194e5](https://github.com/mroonga/mroonga/pull/796/commits/62194e52c96973ef7c1be0886c8a85271387eb66) Sure!